src/Artsy/Router/buildServerApp.tsx

```diff
@@ -95,7 +98,7 @@ export function buildServerApp(config: RouterConfig): Promise<Resolve> {
         status,
       })
     } catch (error) {
-      console.error("[Reaction Router/buildServerApp] Error:", error)
+      console.error("[Artsy/Router/buildServerApp] Error:", error)
```

Comment:
At some point we should look at just having webpack encode the values for us based on `__filename`, so that we don’t need to think about it too much and can just move it around.

Reply:
Ahh this is really interesting, haven't thought about how we can leverage this at build time and inject things right in.